File: trains.client/src/app/models/schedule.ts
import { Time } from "@angular/common";
import { IStation } from "./station";
import { ITrain } from "./train";


export interface ISchedule {
  Id: number;
  Name: string;
  TimeOnStation: Time;
  TimeOutStation: Time;
  StopTime: Time;
  StationId: number;
  Station: IStation;
  Trains: ITrain[];
}

File: Trains.Server/Data/Models/Schedule.cs
namespace Trains.Server.Data.Models
{
    public class Schedule
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public TimeSpan TimeOnStation { get; set; }
        public TimeSpan TimeOutStation { get; set; }
        public TimeSpan StopTime { get; set; }
        public int StationId { get; set; }
        public Station Station { get; set; }
        public List<Train> Trains { get; set; }
    }
}
